src/test/runTest.ts

```diff
@@ -16,12 +17,15 @@ async function main() {
         // The path to test runner
         const extensionTestsPath: string = path.join(__dirname, 'index');
 
+        // Install maven dependencies
+        exec.execSync('mvn clean install', { cwd: path.join(__dirname, 'resources', 'maven', 'multiPomDependency') });
```

Comment:
This is not the correct place to run `mvn install`. 
You can move it to a `before` clause in mavenUtils.test.ts